### PR TITLE
Dataset: TabLib

### PR DESCRIPTION
**Pull request by jpgard**
This PR is submitted in accordance with the Hyak dataset policy described [here](https://hyak.uw.edu/blog/hyak-dataset-policy/).

### CODE REVIEW

File: docs/data-commons/tablib.md
Comment (pnw-ryanmcgr) [inline docs/data-commons/tablib.md]: DAtasets > Datasets

**Review by pnw-ryanmcgr — COMMENTED** [review]
Otherwise, looks great to me.

File: docs/data-commons/tablib.md
Comment (pnw-ryanmcgr) [inline docs/data-commons/tablib.md:6]: Also, you will need to add the following block to the very top of the document
```
---
id: tablib
title: TabLib
---
```

File: docs/data-commons/tablib.md
Comment (jpgard) [reply docs/data-commons/tablib.md]: done.

File: docs/data-commons/tablib.md
Comment (jpgard) [reply docs/data-commons/tablib.md:6]: done.

**Comment by jpgard** [comment]
Thanks for the review @pnw-ryanmcgr !

Let me know what the next steps are, once we have the right levels of approval, so that we can provide any support in getting the actual data hosted!

**Review by pnw-ryanmcgr — APPROVED** [review]

**Pull request merged**
